src/runtime/policy/tests/policy-test.ts

```diff
@@ -359,4 +360,138 @@ policy MyPolicy {
         Capabilities.create([Persistence.inMemory(), new Encryption(false), Ttl.days(2)])),
         `Unexpected capabilities: ${capabilities[1].toDebugString()}`);
   });
+
+  it('restricts types according to policy', async () => {
+    const manifest = await Manifest.parse(`
+schema Address
```

Comment:
For readability can you please indent this Arcs manifest in line with the code and add blank lines in between schemas and policies, ideally:

```
  it('restricts types according to policy', async () => {
    const manifest = await Manifest.parse(`
      schema Address
        number: Number
        street: Text
        city: Text
        country: Text

      schema Person
        name: Text
        phone: Text
        (...)
```

Reply:
Done.